UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -844,8 +844,11 @@ protected virtual async Task ConstructPrimitiveAttributes(MeshPrimitive primitiv
 					Offset = (uint)bufferData.ChunkOffset
 				};
 			}
-
-			GLTFHelpers.BuildMeshAttributes(ref attributeAccessors);
+			try { 
+				GLTFHelpers.BuildMeshAttributes(ref attributeAccessors);
+			} catch (System.Exception e) {
```

Comment:
>System.Exception [](start = 12, length = 16)

As a general rule, you should never catch System.Exception and swallow it.  There are million things it could be and you have no way to know if it's reasonable to continue on.

You can catch the specific one that I suggested you change it to (GLTFLoadException).